lib/file.cpp

```diff
@@ -26,6 +26,17 @@ std::string file$bytes(const std::string &filename)
     return r;
 }
 
+void file$writebytes(const std::string &filename, const std::string &data)
```

Comment:
This function should be moved down so it's declared in the same order as in the neon file (alphabetical order).